src/components/Header.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
```

Comment:
That was not part of the Exercise and should next time be put into a separate commit and organized under the respective issue. But good job getting it done so fast! #Meckernaufhohemniveau